src/main/java/eu/dissco/orchestration/backend/domain/ObjectType.java

```diff
@@ -0,0 +1,20 @@
+package eu.dissco.orchestration.backend.domain;
+
+public enum ObjectType {
+  SOURCE_SYSTEM ("sourceSystem", "https://hdl.handle.net/21.T11148/64396cf36b976ad08267"),
+  MAPPING ("mapping", "https://hdl.handle.net/21.T11148/b3f1045d8524d863ccfb"),
```

Comment:
Why is this fdo type different?

Reply:
To keep sonar happy....  
these are all placeholder fdo profiles right now, but if I have 3 literals, sonar gives a code smell and says to make a constant. I'd have to make a properties file to do that (because it's an enum here), this just was a bit easier.